### PR TITLE
Added full functionality for settings modal screen

### PR DESCRIPTION
**Pull request by ryqui**
code should work a-ok no jons
among-us-twerk

### CODE REVIEW

**Review by hungnguyendinh1999 — APPROVED** [review]
LGTM. Might want @reesque to take a look to be sure

File: frontend/src/screen/SettingScreen.tsx [hunk 1]
```diff
@@ -1,38 +1,413 @@
...
+            }
+
+            const isIndented = key.startsWith("font") && !isSection;
+            return (
+              <div
+                key={key}
+                className={`setting-row ${isIndented ? "indented" : ""}`}
+              >
+                {icon && <img src={icon} className="settings-icon" alt={`${label} Icon`} />}
+                <span>{label}</span>
```
Comment (reesque) [inline frontend/src/screen/SettingScreen.tsx:350]: Change this text color into `var(--main-black)` to match with the rest of the app. I haven't check the buttons, but their label should either be `--main-black` or `--main-white` depends on the background too.

File: frontend/src/assets/background-color-icon.png
Comment (reesque) [inline frontend/src/assets/background-color-icon.png:1]: Use these icons to match the black color

[icons.zip](https://github.com/user-attachments/files/17787281/icons.zip)

**Review by reesque — CHANGES_REQUESTED** [review]
Minor theming change is needed to match the color scheme of other screens

**Comment by reesque** [comment]
Also please remove `DemoVoiceBar` if not needed

**Review by reesque — APPROVED** [review]

**Pull request merged**